Criando arquivo CODE_OF_CONDUCT e CONTRIBUTING (#29)
dev-house-community/ThasfinBot: CONTRIBUTING.md

# CONTRIBUINDO

Aqui você irá encontrar algumas práticas recomendadas que devem ser levadas em consideração na hora de criar **Issues** e **Pull Requests**, e que pode aumentar as chances da sua alteração ser adicionada ao projeto.

> Você verá a importância de descrever de forma correta as alterações feitas, e como isso pode ajudar as pessoas que mantem o projeto e que revisam as **Issues** e **Pull Requests**.

Reviewer: ```suggestion
Você verá a importância de descrever de forma correta as alterações feitas, e como isso pode ajudar as pessoas que mantém o projeto e que revisam as **Issues** e **Pull Requests**.
```

> É muito importate e recomendado que você leia o **[CODE_OF_CONDUCT](CODE_OF_CONDUCT.md)** do projeto antes de qualquer coisa, lá está descrito o que não é permitido de maneira alguma dentro do alcance da comunidade.

Reviewer: ```suggestion
É muito importante e recomendado que você leia o **[CODE_OF_CONDUCT](CODE_OF_CONDUCT.md)** do projeto antes de qualquer coisa, lá está descrito o que não é permitido de maneira alguma dentro do alcance da comunidade.
```

## TÍTULO E DESCRIÇÃO

> Essa é somente uma de várias das prátias recomendadas que devem ser seguidas para criar **Issues** e **Pull Requests** de forma correta, e que pode aumentar as chances de sua modificação ser adicionada ao projeto, além de seguir as recomendações propostas para fazer contrbuições.

Reviewer: ```suggestion
Essa é somente uma de várias das práticas recomendadas que devem ser seguidas para criar **Issues** e **Pull Requests** de forma correta, e que pode aumentar as chances de sua modificação ser adicionada ao projeto, além de seguir as recomendações propostas para fazer contribuições.
```

> Os títulos e descrições precisam ser coerentes e descrever o que realmente está acontecendo. O título deve descrever a intenção da modificação, enquando a descrição deve conter detalhadamente o que foi mudado e a pretenção, isso ajuda com que os revisores possam ter uma visão mais ampla do que você pretende fazer, antes de revisar o código.

Reviewer: ```suggestion
Os títulos e descrições precisam ser coerentes e descrever o que realmente está acontecendo. O título deve descrever a intenção da modificação, enquanto a descrição deve conter detalhadamente o que foi mudado e a pretensão, isso ajuda com que os revisores possam ter uma visão mais ampla do que você pretende fazer, antes de revisar o código.
```

## COMMITS SEMÂNTICOS

> Fazer commits de forma correta também é muito importante em todos os projetos, seja em contribuições **Open Souce**, pojetos privados, ou até mesmo em projetos pessoais, é muito importante ter messagens de commits que descrevam realmente o que a alreração no código está fazendo.

Reviewer: ```suggestion
Fazer commits de forma correta também é muito importante em todos os projetos, seja em contribuições **Open Souce**, projetos privados, ou até mesmo em projetos pessoais, é muito importante ter mensagens de commits que descrevam realmente o que a alteração no código está fazendo.
```

> Em contribuições **Open Souce** uma boa maneira de revisar um **PR** é vendo quais arquivos foram modificados, mas isso muda a partir do momento em que vários arquivos estão sendo modificados, entrando em contextos diferentes, nesses casos navegar atravéz de commits se torna uma melhor opção, podendo analizar o código por contextos de modificação, fica mais compreensível ver o que está acontecendo no código, por isso é muito importante sempre escrever as menssagens de commits de forma clara e legível, além é claro de adicionar somente os arquivos que façam parte do contexto da modificação no commit.

Reviewer: ```suggestion
Em contribuições **Open Souce** uma boa maneira de revisar um **PR** é vendo quais arquivos foram modificados, mas isso muda a partir do momento em que vários arquivos estão sendo modificados, entrando em contextos diferentes, nesses casos navegar através de commits se torna uma melhor opção, podendo analisar o código por contextos de modificação, fica mais compreensível ver o que está acontecendo no código, por isso é muito importante sempre escrever as mensagens de commits de forma clara e legível, além é claro de adicionar somente os arquivos que façam parte do contexto da modificação no commit.
```

> Caso você precise alterar uma menssagem de commit, o Github mostra como fazer isso na documentação no seguinte link. [Alterando menssagens de commits][changing-a-commit-message].

Reviewer: ```suggestion
Caso você precise alterar uma mensagem de commit, o Github mostra como fazer isso na documentação no seguinte link. [Alterando mensagens de commits][changing-a-commit-message].
```

> Depois dessa breve descrição sobre a importancia de ter commits declarativos, podemos entrar no caso em que se possa fazer uma modificação no projeto em grupos ou times, e dar o crédito para todos é muito importante, e para isso você pode adicionar Co-autores aos commits, o Github mostra com o fazer isso. Adicionado Co-autores aos commits - [Criando commits com multiplos autores][commit-with-multiple-authors].

Reviewer: ```suggestion
Depois dessa breve descrição sobre a importância de ter commits declarativos, podemos entrar no caso em que se possa fazer uma modificação no projeto em grupos ou times, e dar o crédito para todos é muito importante, e para isso você pode adicionar Co-autores aos commits, o Github mostra com o fazer isso. Adicionado Co-autores aos commits - [Criando commits com múltiplos autores][commit-with-multiple-authors].
```

Author: O spell check reclama de umas coisas e deixa passar outras, obrigado por revisar isso.
